cf_remote/remote.py

```diff
@@ -255,11 +265,11 @@ def get_info(host, *, users=None, connection=None):
 
 @auto_connect
 def install_package(host, pkg, data, *, connection=None):
-
     print("Installing: '{}' on '{}'".format(pkg, host))
     output = None
+    unix_pkg_path = join("/tmp", pkg)
```

Comment:
`/tmp` is often available to everyone in the system. We would need to make sure it's protected on the target host.

Reply:
yeah, I suppose we could change from assuming the current directory "over there" and instead specify something like `$HOME/.cf-remote-something-tmp`. Will give it a try.